linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_kmod/rule.yml

```diff
@@ -1,4 +1,4 @@
-{{%- if product in ["rhel8"] %}}
+{{%- if product in ["rhel8","rhel7"] %}}
```

Comment:
The RHEL 7 STIG mentions in the manual benchmark the following line for this requirement:

`-w /usr/bin/kmod -p x -F auid!=unset -k module-change`

Note that it's slightly different than RHEL8 STIG although the meaning is probably very similar.